UM/Qt/qml/UM/Slider.qml

```diff
@@ -0,0 +1,71 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.15
+
+import UM 1.7 as UM
+
+Slider
+{
+    id: slider
+    //Draw line
+    background: Rectangle
+    {
+        id: backgroundLine
+        height: UM.Theme.getSize("print_setup_slider_groove").height
```

Comment:
Maybe we should have a `slider_groove` theme key-value and `print_setup_slider_groove` inherits from that.

Same goes for the items entries below.

The naming feels just to specific for the UM component `Slider`

Reply:
The key value only works for colors (I think). I agree the naming is not right, I'll fix that.